test/templates/product-menu/index.njk

```diff
@@ -1,26 +0,0 @@
-{% extends "layout.njk" %}
-{% from "./../format.njk" import format %}
-{% from "../button/component.njk" import button %}
-{% from "./component.njk" import product_menu %}
```

Comment:
If we're removing this, shouldn't we also remove the `data.json` and `component.njk` files in the `product-menu` directory?

Reply:
it is used in shellbar testing